tests/js/list-tests.js

```diff
@@ -31,6 +31,10 @@ const DATE3 = new Date(3);
 const isNodeProcess = typeof process === "object" && process + "" === "[object process]";
 const isElectronProcess = typeof process === "object" && process.versions && process.versions.electron;
 
+global.inspect = function (obj) {
+  console.log("BREAK: ", obj);
+};
```

Comment:
this is not needed anymore

Reply:
oh...i did not meant to commit that.  Good catch